vortex-btrblocks/src/integer.rs

```diff
@@ -401,6 +405,10 @@ impl Scheme for BitPackingScheme {
         if bw as usize == stats.source().ptype().bit_width() {
             return Ok(stats.source().clone().into_array());
         }
+        if bw == 0 {
```

Comment:
I don't think we should even try to compress something with bit-width zero, the compressor should try with Sparse/Constant/etc. instead

Reply:
So my recommendation here would be to return the original array. We don't want to compress with Sparse if we're already downstream of a Sparse anyway

Reply:
I agree, what should happen in this case. I try to stop this in `expected_compression_ratio` but that might miss this, since its a sample correct?